Subject: ieee1394-delete-legacy-module-aliases
From: Jody McIntyre <scjody@steamballoon.com>

amdtp, dv1394, raw1394, video1394:

Delete legacy module aliases.  The macros did not work and the aliases are not
needed nowadays.

Signed-off-by: Stefan Richter <stefanr@s5r6.in-berlin.de>
Signed-off-by: Ben Collins <bcollins@debian.org>
Signed-off-by: Jody McIntyre <scjody@steamballoon.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Olaf Hering <olh@suse.de>
---

 drivers/ieee1394/amdtp.c     |    1 -
 drivers/ieee1394/dv1394.c    |    1 -
 drivers/ieee1394/raw1394.c   |    1 -
 drivers/ieee1394/video1394.c |    1 -
 4 files changed, 4 deletions(-)

diff -puN drivers/ieee1394/amdtp.c~ieee1394-delete-legacy-module-aliases drivers/ieee1394/amdtp.c
--- 25/drivers/ieee1394/amdtp.c~ieee1394-delete-legacy-module-aliases	Thu Sep 22 15:07:29 2005
+++ 25-akpm/drivers/ieee1394/amdtp.c	Thu Sep 22 15:07:29 2005
@@ -1297,4 +1297,3 @@ static void __exit amdtp_exit_module (vo
 
 module_init(amdtp_init_module);
 module_exit(amdtp_exit_module);
-MODULE_ALIAS_CHARDEV(IEEE1394_MAJOR, IEEE1394_MINOR_BLOCK_AMDTP * 16);
diff -puN drivers/ieee1394/dv1394.c~ieee1394-delete-legacy-module-aliases drivers/ieee1394/dv1394.c
--- 25/drivers/ieee1394/dv1394.c~ieee1394-delete-legacy-module-aliases	Thu Sep 22 15:07:29 2005
+++ 25-akpm/drivers/ieee1394/dv1394.c	Thu Sep 22 15:07:29 2005
@@ -2660,4 +2660,3 @@ static int __init dv1394_init_module(voi
 
 module_init(dv1394_init_module);
 module_exit(dv1394_exit_module);
-MODULE_ALIAS_CHARDEV(IEEE1394_MAJOR, IEEE1394_MINOR_BLOCK_DV1394 * 16);
diff -puN drivers/ieee1394/raw1394.c~ieee1394-delete-legacy-module-aliases drivers/ieee1394/raw1394.c
--- 25/drivers/ieee1394/raw1394.c~ieee1394-delete-legacy-module-aliases	Thu Sep 22 15:07:29 2005
+++ 25-akpm/drivers/ieee1394/raw1394.c	Thu Sep 22 15:07:29 2005
@@ -2958,4 +2958,3 @@ static void __exit cleanup_raw1394(void)
 module_init(init_raw1394);
 module_exit(cleanup_raw1394);
 MODULE_LICENSE("GPL");
-MODULE_ALIAS_CHARDEV(IEEE1394_MAJOR, IEEE1394_MINOR_BLOCK_RAW1394 * 16);
diff -puN drivers/ieee1394/video1394.c~ieee1394-delete-legacy-module-aliases drivers/ieee1394/video1394.c
--- 25/drivers/ieee1394/video1394.c~ieee1394-delete-legacy-module-aliases	Thu Sep 22 15:07:29 2005
+++ 25-akpm/drivers/ieee1394/video1394.c	Thu Sep 22 15:07:29 2005
@@ -1571,4 +1571,3 @@ static int __init video1394_init_module 
 
 module_init(video1394_init_module);
 module_exit(video1394_exit_module);
-MODULE_ALIAS_CHARDEV(IEEE1394_MAJOR, IEEE1394_MINOR_BLOCK_VIDEO1394 * 16);
_
